app/models/submission_collector.rb

```diff
@@ -238,20 +238,9 @@ def manually_collect_submission(grouping, rev_num,
     self.safely_stop_child_exited = true
     self.save
 
-    #Let the child process handle conversion, as things go wrong when both
-    #parent and child do this.
-    start_collection_process do
```

Comment:
The call to `start_collection_process` should be kept, as it is what... starts the collection process. I would keep the do block, but remove the checks inside which refer to the unused variables.